web-lib-funcs.pl

```diff
@@ -10697,6 +10706,12 @@ sub close_tempfile
 		chmod($st[2], $_[0]);
 		chown($st[4], $st[5], $_[0]);
 		}
+	if ($file_acls) {
+		# Set original ACLs
+		open(my $pipe, '|-', "$setfacl --restore=-");
```

Comment:
There's an execute_command Webmin function that can be used to feed input from a string to a command

Reply:
Yeah, but I won't use it from inside `close_tempfile`, which can be called hundreds or thousands of times during a call. Using a direct pipe avoids the overhead of forking and will be more performant for this specific case, in my opinion.